src/FunctionCalculation.java

```diff
@@ -0,0 +1,15 @@
+import java.util.function.DoubleUnaryOperator;
+
+public class FunctionCalculation {
+	public static double integrate(DoubleUnaryOperator f, double a, double b) {
+		int n = 10_000_000;
+		double differential = (b - a) * 1.0 / n;
```

Comment:
```suggestion
		double differential = (b - a) / n;
```

Reply:
what about it?

Reply:
I do not know why, but idea uploads changes through time.